app/Entities/Repos/PageRepo.php

```diff
@@ -136,7 +136,13 @@ public function getNewDraftPage(Entity $parent)
             $page->book_id = $parent->id;
         }
 
-        $defaultTemplate = $page->book->defaultTemplate;
+        // check for chapter
+        if ($page->chapter_id) {
+            $defaultTemplate = $page->chapter->defaultTemplate;
+        } else {
+            $defaultTemplate = $page->book->defaultTemplate;
+        }
```

Comment:
This logic presents a breaking change, since currently the book default template option cascades down to apply to pages created in books with that option applied.
With this logic new pages, that are in chapters, won't use the book template if set where the parent chapter does not have its own template assigned.

I think we'd want to retain the existing cascade behaviour of book templates, with a preference of the chapter template if set.

We do have a test to catch this scenario, which is currently failing due to this.

Reply:
If you could also take on this part I would also be happy.